### PR TITLE
release-notes: updates for 20221115 releases

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

**Review by marmijo — APPROVED** [review]
LGTM

File: release-notes/next.yml [hunk 1]
```diff
@@ -33,6 +34,8 @@ releases:
         url: "https://github.com/coreos/fedora-coreos-tracker/issues/1300"
   37.20220910.1.0:
     issues:
+      - text: "tracker: Rebase onto Fedora 37"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1223"
```
Comment (jlebon) [inline release-notes/next.yml:38]: Hmm, that's not new to `next` though.

Comment (dustymabe) [reply release-notes/next.yml:38]: I retroactively added it to the first release of `next` that had F37. Probably should have done that in a separate commit.

Comment (jlebon) [reply release-notes/next.yml:38]: Oh I see, it's an old entry. Gotcha.

**Pull request merged**